packages/meteor-gazelle-ipmanager/lib/bannedIps.js

```diff
@@ -0,0 +1,83 @@
+BannedIps = new Mongo.Collection('bannedIps');
+BannedIp = Astro.Class({
+  name: 'BannedIp',
+  collection: BannedIps,
+  fields: {
+    startIp: 'number',
+    endIp: 'number',
+    startIpStr: 'string',
+    endIpStr: 'string',
+    notes: 'string',
+    expireOn: 'date',
+    createdAt: 'date',
+    createdBy: 'string'
+  },
+  indexes: {
+    startIpEndIp: {
+      fields: {
+        startIp: 1,
+        endIp: 1
+      },
+      options: {}
+    }
+  },
+  validators: {
+    startIp: [
+      Validators.required(),
+      Validators.number()
+
+    ],
+    startIpStr: [
+      Validators.required(),
+      Validators.string()
+    ],
+    endIp: [
+      Validators.required(),
+      Validators.number()
+
+    ],
+    endIpStr: [
+      Validators.required(),
+      Validators.string()
+    ],
+    notes: [
```

Comment:
End ips shouldnt be required. Will fix this. Is this question in regards to notes?